docs/source/getting-started/tabular-data/grouping.ipynb

```diff
@@ -858,10 +858,8 @@
     "\n",
     "query_work_hours = (\n",
     "    context.query()\n",
-    "    .with_columns(HWUSUAL=pl.col.HWUSUAL.cast(int))\n",
-    "    .filter(pl.col(\"HWUSUAL\") != 99)\n",
```

Comment:
Isn't it important to filter out the 99s? Or is that done elsewhere? We don't want them in the mean.

Reply:
It's filtered above, before building the context. It's done that way to preserve the margin metadata